### ISSUE TITLE
Kobo .wpt files issues

### ISSUE THREAD

**Issue by villamany** [LK8000/LK8000#1192]
Hi, im using the lk8000 6.0b in my kobo mini, but im having issues with several .wpt files. Whe i use these files is some cases there are no any waypoint loaded.
Lk8000 Android works fine with the same file.

For example this file is fully waypoits missing in kobo:   https://airtribune-production.s3.amazonaws.com/media/contest/waypoints/2017/06/lap2017.CompeGPS.wpt

I also obseved that when working with "good .wpt files" if i use more than 16 chars for the waypoint name the waypoint is also missing when load the file.

Cheers

**Comment by parapenT1sta** [comment]
Did you try the latest beta version? Your issue might be fixed already: http://www.postfrontal.com/forum/topic.asp?TOPIC_ID=8737

**Comment by AlphaLima** [comment]
have you set "wpt outside terrain" to "include" on Config page 1 ?

**Comment by villamany** [comment]
Yes

**Comment by AlphaLima** [comment]
PLEASE USE THE LK8000.IT FORUM FOR BUG REPORTS!!!

**Comment by brunotl** [comment]
I can't replicate ...

**Comment by villamany** [comment]
Are you using the mini for replicate?

**Comment by brunotl** [comment]
yes

**Comment by villamany** [comment]
I just tried a fresh install again by connecting the kobo to a PC, deleting all files and folders but .kobo directory, copying the "koboRoot.tgz" from the 6.1r to .kobo dir and rebooting, after updating process i copied the linked .wpt file to waypoint directory. But the same result "Waypoints loaded: 0" at startup screen.

**Comment by brunotl** [comment]
check RUNTIME.log ....

**Comment by villamany** [comment]
here is the log after a fresh installation: https://drive.google.com/open?id=1so6iUgdNuHpDH5ghAvYKn9wizj9LDxvs

**Comment by villamany** [comment]
Same issue on the Kobo Aura 2 edition, so i think is a lk8000 related issue.

PD: "Sorry but the lk8000 forum is not working, a lot of issues for registration and logging".

**Comment by brunotl** [comment]
in your Runtime.log is see "[000015988] . NO TERRAIN file available."

so check if you have enabled "Waypoint outside terrain" in system config

**Comment by villamany** [comment]
Yes, this is because the log was after a clean installation without any configuration. But later tested with that option and the terrain file downloaded and same results. 
If i get the same file but translated to .cup format i see all waypoints as expected without touching any configuration.

**Comment by brunotl** [comment]
please send to me your mail,

**Comment by villamany** [comment]
villamany@gmail.com

**Comment by brunotl** [comment]
@parapenT1sta : 

your file is  "GPSDump/FS" wpt file, this format are not supported.

> files with name of .WPT can be different formats.  both OziExplorer, CompeGPS, and GPSDump/FS uses the same .WPT name on the file, but they are in fact different types of files.

Only OziExplorer and CompeGPS ".wpt" files are supported by LK8000...

**Comment by parapenT1sta** [comment]
Sorry! I was using the file given by the organization.
